docs/reference/changelog.rst

```diff
@@ -513,8 +520,8 @@ Services
 - Setting the arguments for a service using the service factory's ``set_kwargs`` is
   deprecated. Use ``update_kwargs`` instead.
 
-Testing
-=======
+Testing with pytest
```

Comment:
This is just so that the pre-commit hooks stop complaining because the `=======` looked like a merge conflict.